pkg/fleet/env/env.go

```diff
@@ -149,7 +150,10 @@ func (e *Env) ToEnv() []string {
 }
 
 func parseApmLibrariesEnv() map[ApmLibLanguage]ApmLibVersion {
-	apmLibraries := os.Getenv(envApmLibraries)
+	apmLibraries, ok := os.LookupEnv(envApmLibraries)
+	if !ok {
+		apmLibraries = os.Getenv(envApmLanguages)
+	}
```

Comment:
we don't support setting both in other scenarios?

Reply:
it's not supported on the s3 install script. envApmLibraries is checked first and then fallsback to envApmLanguages